src/components/Book.js

```diff
@@ -1,8 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { remBook } from '../redux/books/books';
 
 const Book = (props) => {
-  const { title, author } = props;
+  const { title, author, key } = props;
+
+  const dispatch = useDispatch();
+  const interimRemBook = () => {
+    dispatch(remBook(key));
+  };
```

Comment:
- [ ] please make sure you can delete books from your app.

Here is a loom video that explains in more details the issue you are having, I hope it helps.

https://www.loom.com/share/aeba56fcfb9c42158b5d45a582d2227d

Reply:
@andgarzonmal Thanks for the heads up, bro. I appreciate the video as well - I was puzzled as to why it wasn't working as well.